### PR TITLE
Fix/All containers´ performance improvements and fix for varcall_py36 container build

### PR DESCRIPTION
**Pull request by khurrammaqbool**
### This PR:

Fixed: failed container build and push for varcall_py36 and subsequently improve performance of all containers.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Review by ashwini06 — APPROVED** [review]
Faster performance of docker builds, well done 👍

File: BALSAMIC/containers/annotate/annotate.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 
 dependencies:
-  - ensembl-vep=100.2
-  - bcftools>=1.10
-  - vcfanno=0.3.2
-  - gxx_linux-64 
+  - bioconda::ensembl-vep=100.2
+  - bioconda::bcftools>=1.10
+  - bioconda::vcfanno=0.3.2
+  - gxx_linux-64
```
Comment (hassanfa) [inline BALSAMIC/containers/annotate/annotate.yaml:10]: Is it possible to add channel to `gxx_linux-64` and `pip`?

File: BALSAMIC/utils/cli.py [hunk 2]
```diff
@@ -449,7 +449,11 @@ def get_bioinfo_tools_version(bioinfo_tools: dict,
             for p in packages:
                 if isinstance(p, dict):
                     continue
-                name = p.split("=")[0]
+                package = p.split("=")[0]
+                if (len(package.split("::")) > 1):
+                    name = package.split("::")[1]
+                else:
+                    name = package
```
Comment (hassanfa) [inline BALSAMIC/utils/cli.py:456]: adding channel to gxx_lib and pip, will solve this "smelly" code according to SonarCloud :-)

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,26 +4,26 @@
 Added:
 ^^^^^^
 
-* samtools flagstats and stats to workflow and MultiQC
```
Comment (hassanfa) [inline CHANGELOG.rst:7]: Awesome that you also fixed bad formatting on changelog.

**Review by hassanfa — COMMENTED** [review]
Some comments, we'll have to merge in develop and test if they work A-ok! 💯

File: CHANGELOG.rst (hunk 3 above)
Comment (khurrammaqbool) [reply CHANGELOG.rst:7]: Fixed.

File: BALSAMIC/utils/cli.py (hunk 2 above)
Comment (khurrammaqbool) [reply BALSAMIC/utils/cli.py]: Fixed!

**Review by hassanfa — APPROVED** [review]

**Pull request merged**